src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java

```diff
@@ -62,7 +62,7 @@ private AboutDialogContents()
    public AboutDialogContents(ProductInfo info, ProductEditionInfo editionInfo)
    {
       initWidget(uiBinder.createAndBindUi(this));
-      versionLabel.setText(info.version);
+      versionLabel.setText(info.version + "for " + info.os);
```

Comment:
I think making it part of the version number here will emphasize it more than is necessary. Let's put it in the build info along with the release name and commit hash.